File: SharpDevelop/CSharp/w7cpframe/clsPfForce.cs
//
// Copyright (c)2016 S C Harrison
// Refer to License.txt for terms and conditions of use.
//
using System;
using System.Collections.Generic;
using System.Text;
using System.IO;

namespace w7cpframe
{
    class clsPfForce
    {


        public double axial = 0;        //.. axial force ..
        public double shear = 0;        //.. shear force ..
        public double momnt = 0;         //.. end moment ..



        public void initialise()
        {
            axial = 0;
            shear = 0;
            momnt = 0;
        }


        public string sprint()
        {
            string s;

            s = "";
            s = s + axial.ToString().PadLeft( 8);
            s = s + shear.ToString().PadLeft( 8);
            s = s + momnt.ToString().PadLeft( 8);

            return s;

        }



        public void cprint()
        {
            System.Console.WriteLine(sprint());
        }




    } //class
} //name space

File: JScript/clsPfForce.js
//
// Copyright (c)2014 S C Harrison
// Refer to License.txt for terms and conditions of use.
//


//Define Class
function clsPfForce()
{

  axial = 0;        //.. axial force ..
  shear = 0;        //.. shear force ..
  momnt =0;         //.. end moment ..
}


clsPfForce.prototype.initialise = function()
{
  this.axial = 0;
  this.shear = 0;
  this.momnt = 0;
}


clsPfForce.prototype.sprint = function()
{
  var s;
  
  s = "";
  s = s + StrLPad(this.axial.toString(), 8);
  s = s + StrLPad(this.shear.toString(), 8);
  s = s + StrLPad(this.momnt.toString(), 8);

return s;

}



clsPfForce.prototype.cprint = function()
{
  WScript.Echo(this.sprint());
}

